arkouda/numpy/_numeric.py

```diff
@@ -1296,18 +1311,16 @@ def _trig_helper(pda: pdarray, func: str, where: Union[bool, pdarray] = True) ->
     Raises
     ------
     TypeError
-        Raised if the parameter is not a pdarray
-    TypeError
-        Raised if where condition is not type Boolean
+        Raised if pda is not a pdarray or if is not real or int or uint, or if where is not Boolean
     """
+    datatype_check(pda.dtype, [float, int, ak_uint64], func)
```

Comment:
I don't think it will affect the behavior, but I think we should be using `ak.float64` and `ak.int64` rather than the python built-ins for data type checking.

Reply:
Since they match, it won't, but I agree as a matter of look-and-feel.  I'll make that change.